test/cri-containerd/helper_exec_test.go

```diff
@@ -78,12 +78,12 @@ func execInHost(ctx context.Context, client shimdiag.ShimDiagService, args []str
 }
 
 // shimDiagExecOutput is a small wrapper on top of execInHost, that returns the exec output
-func shimDiagExecOutput(ctx context.Context, tb testing.TB, podID string, cmd []string) string {
-	tb.Helper()
+func shimDiagExecOutput(ctx context.Context, t testing.TB, podID string, cmd []string) (string, error) {
```

Comment:
can we split `shimDiagExecOutput` into two so we dont have to update all the call sites and repeate `if err != nil { t.Fatal(err) }` everywhere?

```go
func shimDiagExecOutput(ctx context.Context, tb testing.TB, podID string, cmd []string) string {
    tb.Helper()
    out, err := shimDiagExecOutputErr(ctx, podID, cmd)
    if err != nil {
        tb.Fatal(err)
    }
    return out
}

func shimDiagExecOutputErr(ctx context.Context, podID string, cmd []string) (string, error) {
    // ...
}
```